test/generate-inputs/expected/localneighborhood-network-expected.txt

```diff
@@ -0,0 +1,2 @@
+A|B
```

Comment:
Like the other test files, this should have `test_A` as a node. `test_generate_inputs.py` is also missing `localneighborhood` in its `algo_exp_file` dictionary, making `test_generate_inputs` falsely pass.